easybuild/easyconfigs/p/ParaView/ParaView-PythonSix-Update.patch

```diff
@@ -0,0 +1,533 @@
+Update for VTK third party dependency PythonSix(https://pypi.python.org/pypi/six)
+From version 1.7.2 to version 1.11.0
```

Comment:
Ugh, there must be a better way, I sure hope we can avoid this...

Can't we just avoid that ParaView vendors `six`, it's already there in the `Python` we provide...

Reply:
i know its a really bad way to do it..
i just didnt come up with anything else.
Would be cool if you had other ideas to do it

Reply:
im not sure but it seems like you can delete the six.py file in the paraview dir and paraview will detect the python lib instead;
is there a way to set (like a configopt ) to not copy a specific file?